src/metadata/migration/1648476863240-init.ts

```diff
@@ -1,10 +1,10 @@
 import {MigrationInterface, QueryRunner} from "typeorm";
 
-export class init1647661129900 implements MigrationInterface {
-    name = 'init1647661129900'
+export class init1648476863240 implements MigrationInterface {
+    name = 'init1648476863240'
 
     public async up(queryRunner: QueryRunner): Promise<void> {
-        await queryRunner.query(`CREATE TABLE "iasql_database" ("pg_name" character varying NOT NULL, "alias" character varying NOT NULL, "region" character varying NOT NULL, "pg_user" character varying NOT NULL, CONSTRAINT "PK_a70daf5038ee78e1a4e5cb78990" PRIMARY KEY ("pg_name"))`);
+        await queryRunner.query(`CREATE TABLE "iasql_database" ("pg_name" character varying NOT NULL, "alias" character varying NOT NULL, "region" character varying NOT NULL, "pg_user" character varying NOT NULL, "is_ready" boolean NOT NULL DEFAULT true, CONSTRAINT "PK_a70daf5038ee78e1a4e5cb78990" PRIMARY KEY ("pg_name"))`);
```

Comment:
Next think: how are we going to get this change applied to the staging and prod metadata DBs? They persist between deploys, so we should probably migrate them the normal way?

Reply:
Oh that's true. So I can create the migration and then we migrate them manually once landed, right?

Reply:
Yeah, or we can update our own deploy script to always run the migration after `iasql_apply`ing the new Fargate config?

Reply:
oh this actually treats migrations as inmutable in the more traditional sense. we should do a 2nd / additional migration and not modify the existing one like we do everywhere else. If we modify the existing migration typeorm might barf on a file checksum difference unless we blow up the table (and it's contents). we run all migrations on server startup so it should just work because it would recognize that it needs to run the 2nd migration

Reply:
sounds good